zubbi/models.py

```diff
@@ -193,9 +200,9 @@ def _select_doc_type(doc_type, block_class):
             return [block_class]
         return [AnsibleRole, ZuulJob]
 
-    def search_query(self, query, fields, exact=False, extra_filter=None):
-        # Elasticsearch cannot work with a set, only with list
```

Comment:
Maybe it's worth to keep the comment as we still change the value to a list ;-)

Reply:
will fix.